projects/plugins/social/src/js/components/panel/index.jsx

```diff
@@ -71,6 +83,7 @@ const PublicizePanel = ( { prePublish } ) => {
 				<PublicizeForm
 					isPublicizeEnabled={ isPublicizeEnabled }
 					isRePublicizeFeatureEnabled={ ! isPostPublished }
+					numberOfSharesRemaining={ isShareLimitEnabled ? numberOfSharesRemaining : undefined }
```

Comment:
Nitpick: Should we pass`null` instead of `undefined`?

Reply:
Yes, good idea. It'll be consistent with the default value in the `PublicizeForm` component.